material-intro-screen/src/main/java/agency/tango/materialintroscreen/listeners/scroll/ParallaxScrollListener.java

```diff
@@ -16,7 +16,7 @@ public ParallaxScrollListener(SlidesAdapter adapter) {
         this.adapter = adapter;
     }
 
-    @SuppressWarnings("ConstantConditions")
+    @SuppressWarnings({"ConstantConditions", "PointlessNullCheck"})
```

Comment:
there is a bug in this method 
checking  `fragment instanceof Parallaxable `2 times

Reply:
These are two separate if statements. That's why we need to check it twice.